examples/demo-nft-module/README.md

```diff
@@ -0,0 +1,481 @@
+# How to create new module for Sovereign Rollup
+
+
+# Getting Started
+
+Module is a separate crate that needs to implement `sov_modules_api::Module` trait and it's own change logic based on input messages.
+
+Let's start with defining basic dependencies in `Cargo.toml`:
+
+```toml
+[dependencies]
+anyhow = { anyhow = "1.0.62" }
+sov-modules-api = { git = "https://github.com/Sovereign-Labs/sovereign.git", branch = "main", default-features = false }
+sov-modules-macros = { git = "https://github.com/Sovereign-Labs/sovereign.git", branch = "main" }
+```
+
+## Module definition
+
+NFT module is defined as following:
```

Comment:
This feels a little surprising - we haven't told readers _that_ we're building an NFT module yet, but we jump right into telling them how to do it. 

We also haven't explained where this code should live, or what the directory structure should look like. Maybe we should start by telling them that we'll follow the structure of the module template (with a link to the template that they can clone), and then we can add a note saying that this code replaces the `ExampleStruct` in `lib.rs`